kitchen.yml

```
@@ -71,3 +71,9 @@ suites:
   - name: vdir
     run_list:
       - recipe[test::vdir]
+  - name: default-windowsfeatures-powershell
+    run_list:
+      -recipe[iis::default]
```

you need a space in your yaml 
```suggestion
      - recipe[iis::default]
```